commands/faas.go

```diff
@@ -88,3 +84,10 @@ func runFaas(cmd *cobra.Command, args []string) {
 	fmt.Printf(figletStr)
 	cmd.Help()
 }
+
+func capitaliseFirst(str string) string {
+	for _, c := range str {
```

Comment:
Is there nothing in `strings` that can do this? Is there a normal way of printing errors in Golang that we can tap into?

Reply:
See https://github.com/golang/go/wiki/CodeReviewComments#error-strings, error strings are normally printed as-is but after some other text ie "Error: %s" type of thing, which is not how they're being used in the CLI at this point.